### PR TITLE
Switch ARM template to Bicep

### PR DESCRIPTION
**Pull request by johndowns**
This PR updates the example to use Bicep instead of an ARM template.

### CODE REVIEW

File: resources-stamp.bicep [hunk 1]
```diff
@@ -0,0 +1,240 @@
+@description('DNS label prefix: <prefix>.<region>.cloudapp.azure.com')
+param subdomainName string
+
+@description('Domain region: <prefix>.<region>.cloudapp.azure.com')
+param location string
+
+@description('Existing Public IP resource ID or \'newIp\' to indicate an IP address should be created.')
+param ipResourceId string = 'newIp'
```
Comment (ferantivero) [inline resources-stamp.bicep:8]: Is this being used when generating the second cert? Not a big fan of this approach. if this is not a well-known practice, I would rather always  create. Thoughts?

**Review by ferantivero — APPROVED** [review]
nicely done @johndowns, a single comment not strictly related to this PR

File: resources-stamp.bicep (hunk 1 above)
Comment (johndowns) [reply resources-stamp.bicep:8]: I agree with out, but I'd like to check with @ckittel about the original intention here. I will make a note to look at this separately.

**Comment by ferantivero** [comment]
#sign-off

**Pull request merged**